CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+<h1 style="border-bottom: 0;">Contribution Guideline</h1>
```

Comment:
FYI: It's probably expected for this file to be modified after completion of all major refactors inside the code